### ISSUE TITLE
PRINT exception: com.fasterxml.jackson.databind.node.ArrayNode cannot be cast to com.fasterxml.jackson.databind.node.ObjectNode

### ISSUE THREAD

**Issue by rmoff** [confluentinc/ksql#2847]
Topic data: 

```
$ kafkacat -b localhost:9092 -t networkrail_TD_LNE_GN_SIG_AREA-03
% Auto-selecting Consumer mode (use -P or -C to override)
[{"CA_MSG":{"to":"0300","time":"1558429736000","area_id":"DR","msg_type":"CA","from":"0812","descr":"1E02"}},{"CA_MSG":{"to":"0052","time":"1558429740000","area_id":"DR","msg_type":"CA","from":"0056","descr":"1A59"}},{"CC_MSG":{"to":"COUT","time":"1558429740000","area_id":"SC","msg_type":"CC","descr":"2P07"}},{"CA_MSG":{"to":"0606","time":"1558429740000","area_id":"DR","msg_type":"CA","from":"0608","descr":"2P07"}},{"CC_MSG":{"to":"COUT","time":"1558429740000","area_id":"SC","msg_type":"CC","descr":"2P07"}},{"CA_MSG":{"to":"0391","time":"1558429740000","area_id":"PB","msg_type":"CA","from":"0389","descr":"*LB*"}},{"CA_MSG":{"to":"0858","time":"1558429742000","area_id":"DR","msg_type":"CA","from":"0860","descr":"1E03"}}]
```

KSQL 5.2.1 error: 

```
ksql> PRINT 'networkrail_TD_LNE_GN_SIG_AREA-03';
com.fasterxml.jackson.databind.node.ArrayNode cannot be cast to com.fasterxml.jackson.databind.node.ObjectNode
ksql>
```

**Comment by rmoff** [comment]
dup of #1933
